packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
+
+-   We moved external dependencies to devDependencies as it is more logical since those packages are not bundled by Rollup. This change also fix a [Server-side Request Forgery](https://security.snyk.io/vuln/SNYK-JS-IP-6240864) vulnerability reported by Snyk.
```

Comment:
Should we perhaps write those as separate bullet points? My suggestion:

- We moved Pluggable Widgets Tools dependencies marked as external in rollup configuration files to devDependencies in the `package.json`. (We don't need to explain the reasoning behind it, perhaps? I leave this one to you.)

- We fixed a [Server-side Request Forgery](https://security.snyk.io/vuln/SNYK-JS-IP-6240864) vulnerability reported by Snyk.

- We synced the version of `@types/react` package.

- We synced the versions of resolution and override packages

Reply:
Agree, fixed! ❇️